packages/codegen-ui-react/lib/forms/form-renderer-helper/value-props.ts

```diff
@@ -74,10 +74,10 @@ export const renderDefaultValueAttribute = (
     expression = factory.createJsxExpression(undefined, convertedValueAttributeMap[dataType](identifier));
   }
 
-  const componentsWithValue: string[] = ['TextField', 'TextAreaField'];
+  const componentsWithValue = new Set(['TextField', 'TextAreaField']);
```

Comment:
Since this has only two values, a regular old `if` statement is probably cool too.

Reply:
Makes sense. pushing another rev